production/cm_tools/setup.py

```diff
@@ -23,7 +23,8 @@
     "typing",
     "geopandas",
     "gdal",
-    "rsgislib",
+    # The absurdity of rsgislib is that it's installed and it's not able to be checked
+    # "rsgislib",
```

Comment:
still it is project dependancy so don't you keep in requirement list?

Reply:
Thing is, you can't install it using Pip, so including it in here actually stops the build.

Reply:
Maybe it needs a better note.